packages/pn-personafisica-webapp/src/navigation/__test__/navigation.utility.test.ts

```diff
@@ -1,19 +1,20 @@
 import { vi } from 'vitest';
+
 import { EventPageType } from '@pagopa-pn/pn-commons';
 
 import { getConfiguration } from '../../services/configuration.service';
 import { getCurrentEventTypePage, goToLoginPortal } from '../navigation.utility';
 import { APP_STATUS, DELEGHE, DETTAGLIO_NOTIFICA, NOTIFICHE, RECAPITI } from '../routes.const';
 
-const replaceFn = vi.fn();
+const mockOpenFn = vi.fn();
 
 describe('Tests navigation utility methods', () => {
   const original = window.location;
 
   beforeAll(() => {
-    Object.defineProperty(window, 'location', {
-      writable: true,
-      value: { href: '', replace: replaceFn },
+    Object.defineProperty(window, 'open', {
```

Comment:
rewrite the code in afterAll as for Footer.test.tsx. do it for all the test updated